sdk/src/wallet/utxo.ts

```diff
@@ -113,7 +126,8 @@ export async function createBitcoinPsbt(
                 addressInfo.type,
                 publicKey
             );
-            possibleInputs.push(input);
+            // to support taproot addresses we want to exclude outputs which contain inscriptions
+            if (!outpointsSet.has(OutPoint.toString(utxo))) possibleInputs.push(input);
```

Comment:
with `cardinal` argument:
> return UTXOs **not** containing inscriptions or runes

Hence the condition should be flipped here (i.e. remove the `!`)